webapp-django/crashstats/crashstats/models.py

```diff
@@ -1085,19 +1086,14 @@ class CrontabberState(SocorroMiddleware):
     API_WHITELIST = None
 
 
-class BugzillaAPI(SocorroCommon):
-    base_url = settings.BZAPI_BASE_URL
-    username = password = None
-
-
-class BugzillaBugInfo(BugzillaAPI):
-
-    # This is for the whole model.
-    cache_seconds = 0
+class BugzillaBugInfo(SocorroCommon):
 
     # This is for how long we cache the metadata of each individual bug.
     BUG_CACHE_SECONDS = 60 * 60
 
+    # How patient we are with the Bugzilla REST API
+    BUGZILLA_REST_TIMEOUT = 5  # seconds
```

Comment:
I *think* this is too short a timeout for bugzilla. Maybe bugzilla is not as slow as it used to be nowadays, but I don't think it would hurt to increase this a bit, especially since we query this API via AJAX. It is not a blocking resource.